cmd/root.go

```diff
@@ -79,9 +77,22 @@ func getLatestReleaseInfo(info chan<- string) {
 
 func getOcClient() *occlient.Client {
 	client, err := occlient.New()
+	checkError(err, "")
+	return client
+}
+
+// checkError prints the cause of the given error and exits the code with an
+// exit code of 1.
+// If the context is provided, then that is printed, if not, then the cause is
+// detected using errors.Cause(err)
+func checkError(err error, context string) {
 	if err != nil {
-		fmt.Println(err)
+		if context == "" {
```

Comment:
add `log.Debugf("ERROR: %s", err)`and it is perfect

Reply:
done